xml/cl.xml

```diff
@@ -4836,9 +4836,9 @@ server's OpenCL/api-docs repository.
                 <type name="CL/cl.h"/>
             </require>
             <require comment="cl_sampler_properties">
-                <enum name="CL_SAMPLER_MIP_FILTER_MODE"/>
-                <enum name="CL_SAMPLER_LOD_MIN"/>
-                <enum name="CL_SAMPLER_LOD_MAX"/>
+                <enum name="CL_SAMPLER_MIP_FILTER_MODE_KHR"/>
+                <enum name="CL_SAMPLER_LOD_MIN_KHR"/>
+                <enum name="CL_SAMPLER_LOD_MAX_KHR"/>
```

Comment:
Thanks - yes, this is exactly what I had in mind.

The last thing to change is to add these suffixed enums as aliases for the un-suffixed enums in the `<enums>` section.  Something like:

```xml
        <enum value="0x1156"        name="CL_SAMPLER_LOD_MIN"/>
        <enum value="0x1156"        name="CL_SAMPLER_LOD_MIN_KHR"/>
```

Reply:
Thanks for your feedback, have updated the `<enums>` section as well. https://github.com/KhronosGroup/OpenCL-Docs/pull/252/commits/f1c9cba11a7bc1dcea327bd4a16071179402b7b7#diff-62465512a1e1a33f8cdaee21f4a5486cR1293-R1297